### PR TITLE
Ebnt 186 aiohttp server

### PR DESCRIPTION
**Pull request by i-a-andrianov**

### CODE REVIEW

File: examples/binary_payload/serve.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def add_alpha(img):
 
 
 #  This is the overlay we will put on top of provided image
-OVERLAY = imread('ebaklya.png')
+OVERLAY = imread('examples/binary_payload/ebaklya.png')
```
Comment (mike0sv) [inline examples/binary_payload/serve.py]: i think this examples are supposed to run from their respective directories. maybe at least make this path relative to `__file__`?

File: src/ebonite/runtime/server/base.py [hunk 2]
```diff
@@ -50,3 +54,65 @@ def start(self, loader: InterfaceLoader):
...
+    server health check - via HTTP GET call to `/health`.
+
+    Host to which server binds is configured via `EBONITE_HOST` environment variable:
+    default is `0.0.0.0` which means any local or remote, for rejecting remote connections use `localhost` instead.
+
+    Port to which server binds to is configured via `EBONITE_PORT` environment variable: default is 9000.
+    """
+
+    @staticmethod
+    def _deserialize_json(interface: Interface, method: str, request_json: dict):
```
Comment (mike0sv) [inline src/ebonite/runtime/server/base.py]: why private method?

File: examples/binary_payload/serve.py (hunk 1 above)
Comment (i-a-andrianov) [reply examples/binary_payload/serve.py]: Running examples from their directories is difficult. When running we should specify PYTHONPATH for Ebonite sources. If we are in example directory this leads to `../../src`-like things, moreover we should move to each directory if we want to consequtively run several examples.

File: src/ebonite/runtime/server/base.py (hunk 2 above)
Comment (i-a-andrianov) [reply src/ebonite/runtime/server/base.py]: uhm, this method is protected as it is expected to be run by subclasses only, not by clients.

File: examples/binary_payload/serve.py (hunk 1 above)
Comment (mike0sv) [reply examples/binary_payload/serve.py]: i thought that you run examples with legitimately installed ebonite (if you are not a developer). If you are a developer, this is resolved by IDE settings

File: src/ebonite/runtime/server/base.py (hunk 2 above)
Comment (mike0sv) [reply src/ebonite/runtime/server/base.py]: but you call it not from subclasses in flask and aiohttp code

Comment (i-a-andrianov) [reply src/ebonite/runtime/server/base.py]: Yes but it is called from methods which are essentially implementation details of corresponding servers. Ideally these methods should be moved to servers but I decided not to bother with this as it makes no real difference.

File: examples/binary_payload/serve.py (hunk 1 above)
Comment (i-a-andrianov) [reply examples/binary_payload/serve.py]: ok fixed

**Pull request merged**